try {

    throw 'error'
} 
catch {
    console.log('je error') 
}
console.log('pes')